azurecompute/src/main/java/org/jclouds/azurecompute/binders/DeploymentParamsToXML.java

```diff
@@ -50,21 +50,11 @@
                     .e("AdminPassword").t(params.password()).up()
                     .e("ResetPasswordOnFirstLogon").t("false").up()
                     .e("EnableAutomaticUpdate").t("false").up()
-                    .e("DomainJoin")
-                    .e("Credentials")
-                    .e("Domain").t(params.name()).up()
-                    .e("Username").t(params.username()).up()
-                    .e("Password").t(params.password()).up()
-                    .up() // Credentials
-                    .e("JoinDomain").t(params.name()).up()
-                    .up() // Domain Join
-                    .e("StoredCertificateSettings").up()
```

Comment:
@bostko Perhaps make them optional so we can still join a domain if we need to (for example when testing kerberos)?
I assume it's a NOP right now as the domain name doesn't exist.